dashboard/db/migrate/20211217193721_create_reference_guides.rb

```diff
@@ -0,0 +1,13 @@
+class CreateReferenceGuides < ActiveRecord::Migration[5.2]
```

Comment:
As a general rule, we like to split migrations out from other changes. This PR overall seems safe to bundle but there is a lot of logic in here (including seeding and serializing). I'm wondering if it would be easier to review if this PR was split? I don't care that strongly though, so if it's too difficult, I'm okay with keeping this all in one PR